### DISCUSSION TITLE
Drehen der Anzeige in Laufrichtung (seitenrichtig zur Realität)

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#2481] [Ideas]
Ich würde mich wünschen, dass es möglich wäre, dass die Karte sich automatisch beim Laufen so mitdreht wie man läuft, d.h. dass links/rechts und vorne/hinten so dargestellt werden, wie man läuft.

**Comment by FloEdelmann** [comment]
@OpenGreenStreet Ich glaube, du solltest versuchen, auf Englisch zu schreiben, damit deine Frage möglichst viele Leute verstehen und dir antworten können. Der Hauptentwickler @westnordost kann dir aber auch auf Deutsch antworten.

Um auf deine Frage zu antworten: Diesen Modus gibt es schon; probier mal, auf das Kompass-Symbol zu tippen. Erst kürtzlich wurde dieser Modus verbessert: https://github.com/streetcomplete/StreetComplete/issues/2465 (ist aber noch nicht released).

**Reply by OpenGreenStreet** [reply]
@FloEdelmann, super vielen Dank für den Hinweis. Habe es gearde ausprobiert und es funktioniert prima. Danke auch für den Hinweis zur Sprache. Ich versuche, daran zu denken. Viele Grüße OpenGreenStreet
